### PR TITLE
JD editorial suggestions (Weeks 7 and 8)

### PR DESCRIPTION
**Pull request by jdarch**

### CODE REVIEW

File: content/07. Neural networks/.md/07.2 Multidimensional data and the MLP.md [hunk 1]
```diff
@@ -263,7 +266,7 @@ We can now preview the size of the network that has been created:
 fruit.n_features_in_,  fruit.n_layers_, fruit.n_outputs_, fruit.hidden_layer_sizes
 ```
 
-We can import a convenience function that will present this as a simple report:
+We can import a convenient function that will present this as a simple report:
```
Comment (psychemedia) [inline content/07. Neural networks/.md/07.2 Multidimensional data and the MLP.md:269]: No, these really are called convenience functions...

File: content/07. Neural networks/.md/07.1 Introducing neural networks.md [hunk 2]
```diff
@@ -101,11 +101,11 @@ A process known as ‘transfer learning’ allows a model trained on one set of
 
 The pre-trained model already knows how to identify lots of different unique ‘features’ that might be contained within an image. These features may be quite abstract; for example, the network might be able to recognise straight lines, or right angles, or several distinguishable points positioned in a particular way relative to each other, or other patterns that defy explanation (to us, at least).
 
-When you further train the model, it uses combinations of the features it can already detect as components of different feature collections that it can use to distinguish between the different object types you train it to recognise specifically.
+When you further train the model, it uses combinations of the features it can already detect as components of different feature collections that it can use to distinguish between the different object types you train it to recognise specifically.<span style="color:red">JD: I find this sentence confusing. Can it be simplified?</span>
```
Comment (psychemedia) [inline content/07. Neural networks/.md/07.1 Introducing neural networks.md:104]: Changed to: "When transfer learning is used to further train the model, combinations of the features it can already detect are used to create new feature combinations. The new combinations are better able to identify patterns in the specific image collections used to "top-up" the training of the network."

File: content/07. Neural networks/.md/07.2 Multidimensional data and the MLP.md (hunk 1 above)
Comment (jdarch) [reply content/07. Neural networks/.md/07.2 Multidimensional data and the MLP.md:269]: Ah, sorry.

File: content/07. Neural networks/.md/07.1 Introducing neural networks.md (hunk 2 above)
Comment (jdarch) [reply content/07. Neural networks/.md/07.1 Introducing neural networks.md:104]: Thanks (although ‘top up’ with single, curly quotes, if possible.

File: content/07. Neural networks/.md/07.2 Multidimensional data and the MLP.md [hunk 3]
```diff
@@ -150,8 +152,9 @@ In these cases, the MLP uses a *winner-takes-all* strategy in which the largest
...
 
-If the second output identifies the *pear* class, then for the input `(9.5, 1.9)` the desired output would be `(0, 1, 0, 0)`. The values `(9.5, 1.9)` and `(0, 1, 0, 0)` could then be used as a ‘training pair’ of known inputs and outputs.
+If the second output identifies the *pear* class, then for the input `(9.5, 1.9)` <span style="color:red">JD: earlier, (9.5, 1.9) was a bananna. Perhaps change this to e.g. (5.1, 2.1)</span> the desired output would be `(0, 1, 0, 0)`. The values `(9.5, 1.9)` <span style="color:red">JD: ? (5.1, 2.1) </span>and `(0, 1, 0, 0)` could then be used as a ‘training pair’ of known inputs and outputs.
 
+<span style="color:red">JD: Or, change the image so that both images have the same ordering of fruit, keep (9.5, 1.9) and change the start of the previous paragraph to 'If the second output identifies the *banana* class, ... </span>
```
Comment (psychemedia) [inline content/07. Neural networks/.md/07.2 Multidimensional data and the MLP.md:157]: I changes the image; couldn't get Grpahivz to respect my ordering though so manually had to fudge the labels by eye..

File: content/07. Neural networks/.md/07.1 Introducing neural networks.md (hunk 2 above)
Comment (psychemedia) [reply content/07. Neural networks/.md/07.1 Introducing neural networks.md:104]: I've left a lot fo the double quotes, rather than change to single quotes as idiomatic (or maybe idiosyncratic). Why do you prefer the single quote?

Comment (jdarch) [reply content/07. Neural networks/.md/07.1 Introducing neural networks.md:104]: >> Why do you prefer the single quote?
I don't think I do actually prefer single quotes (because using double quotes for quotes means that the single quote mark character is reserved for apostrophes). But single quotes is OU House Style.
